app/assets/javascripts/utils/course_utils.js

```diff
@@ -93,6 +93,19 @@ export default class CourseUtils {
       };
     }
 
+    const wikiRedLinkUrlParts = /([a-z-]+)\.(?:m\.)?(wik[a-z]+)\.org\/w\/index\.php\?title=([\w%]*)[^a-zA-Z0-9%](?:[^#]*)/.exec(articleTitle);
```

Comment:
This is a more general URL format than just redlinks. `/w/index.php?title=` can display any wiki page, equivalently to `/wiki/`.

I suggest changing the name of this constant to reflect that this code is for matching any wiki article URL that uses the `index.php` format.